flink/v1.18/flink/src/test/java/org/apache/iceberg/flink/MiniClusterResource.java

```diff
@@ -50,4 +51,18 @@ public static MiniClusterWithClientResource createWithClassloaderCheckDisabled()
             .setConfiguration(DISABLE_CLASSLOADER_CHECK_CONFIG)
             .build());
   }
+
+  public static MiniClusterWithClientResource createWithClassloaderCheckDisabled(
```

Comment:
There's an ongoing process to move from junit4 to junit5 tests. It would be good to add new features to the junit5 tests only

Reply:
in this case, there is no new test class added. maybe junit5 should be handled separately?

Reply:
I am concerned that we are adding new feature to the old testing harness here

Reply:
See comments below